xgo-scripts/packetbeat_before_build.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/sh
 
-PB_PATH=/go/src/github.com/elastic/packetbeat
+PB_PATH=/go/src/github.com/elastic/beats/
```

Comment:
I just did a find/replace on #37.

But functionally it is correct (the variable might not be named so nicely). The patch needs to be applied from the beats directory because it affects the vendored code.